extensions/amp-accordion/0.1/amp-accordion.js

```diff
@@ -335,6 +343,10 @@ class AmpAccordion extends AMP.BaseElement {
     }
     this.currentState_[contentId] = !isSectionClosedAfterClick;
     this.setSessionState_();
+
+    // Take the records to clear them out. This prevents mutations from
+    // the truncation from invoking the observer's callback.
```

Comment:
Can you help me understand what this comment is referring to? I haven't heard of truncation happening in an amp-accordion

Reply:
Haha oops, this comment is an artifact of taking from another example.

Reply:
I'm going to get rid of this line entirely since there aren't any queued DOM changes (so `takeRecords` here almost always just returns `[]`).